manifest.xml

```diff
@@ -0,0 +1,13 @@
+<package>
+    <description brief="Orocos soem_robotiq_drivers Component package">
+        This package contains the components of the rtt_soem_robotiq package
+    </description>
+    <!--NOTE: set the license and author before you publish this code-->
+    <license>LGPL</license>
+    <author>Wilson Ko</author>
```

Comment:
You're not really the author of this code ;) Maybe add yourself as a maintainer and put the original author in here?